inst/rmarkdown/templates/informs/resources/template.tex

```diff
@@ -0,0 +1,202 @@
+  \ifodd #1
+  \let\oldpar\par
+  \def\par{\hangindent=\cslhangindent\oldpar}
+  \fi
+  % set entry spacing
+  \setlength{\parskip}{#2\cslentryspacingunit}
+ }%
+ {}
+\usepackage{calc}
+\newcommand{\CSLBlock}[1]{#1\hfill\break}
+\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
+\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
+\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
+$endif$
+
+$for(header-includes)$
+$header-includes$
+$endfor$
+
+%% END MY ADDITIONS %%
+
+
+% Natbib setup for author-year style
+\usepackage{natbib}
+ \bibpunct[, ]{(}{)}{,}{a}{}{,}%
+ \def\bibfont{\small}%
+ \def\bibsep{\smallskipamount}%
+ \def\bibhang{24pt}%
+ \def\newblock{\ }%
+ \def\BIBand{and}%
+
+
+%% Setup of theorem styles. Outcomment only one.
+%% Preferred default is the first option.
+\TheoremsNumberedThrough     % Preferred (Theorem 1, Lemma 1, Theorem 2)
+%\TheoremsNumberedByChapter  % (Theorem 1.1, Lema 1.1, Theorem 1.2)
+\ECRepeatTheorems
+
+%% Setup of the equation numbering system. Outcomment only one.
+%% Preferred default is the first option.
+\EquationsNumberedThrough    % Default: (1), (2), ...
+%\EquationsNumberedBySection % (1.1), (1.2), ...
+
+% For new submissions, leave this number blank.
+% For revisions, input the manuscript number assigned by the on-line
+% system along with a suffix ".Rx" where x is the revision number.
+\MANUSCRIPTNO{$manuscriptno$}
+
+%%%%%%%%%%%%%%%%
+\begin{document}
+%%%%%%%%%%%%%%%%
+
+% Outcomment only when entries are known. Otherwise leave as is and
+%   default values will be used.
+%\setcounter{page}{1}
+%\VOLUME{00}%
+%\NO{0}%
+%\MONTH{Xxxxx}% (month or a similar seasonal id)
+%\YEAR{0000}% e.g., 2005
+%\FIRSTPAGE{000}%
+%\LASTPAGE{000}%
+%\SHORTYEAR{00}% shortened year (two-digit)
+%\ISSUE{0000} %
+%\LONGFIRSTPAGE{0001} %
+%\DOI{10.1287/xxxx.0000.0000}%
+
+% Author's names for the running heads
+% Sample depending on the number of authors;
+% \RUNAUTHOR{Jones}
+% \RUNAUTHOR{Jones and Wilson}
+% \RUNAUTHOR{Jones, Miller, and Wilson}
+% \RUNAUTHOR{Jones et al.} % for four or more authors
+% Enter authors following the given pattern:
+%\RUNAUTHOR{}
+
+$if(short_title)$
+\RUNTITLE{$short_title$}
+$else$
+\RUNTITLE{$title$}
+$endif$
+
+\TITLE{$title$}
+
+\ARTICLEAUTHORS{%
+$for(authors)$
+\AUTHOR{$authors.name$}
+\AFF{$authors.affiliation$, \EMAIL{$authors.email$}}
+
+$endfor$%
+}
+
+\ABSTRACT{$abstract$}
+
+\KEYWORDS{$for(keywords)$$keywords$$sep$; $endfor$}
+
+\maketitle
+
+$for(include-before)$
+$include-before$
+
+$endfor$
+
+$body$
+
+% Appendix here
+% Options are (1) APPENDIX (with or without general title) or
+%             (2) APPENDICES (if it has more than one unrelated sections)
+% Outcomment the appropriate case if necessary
+%
+% \begin{APPENDIX}{<Title of the Appendix>}
+% \end{APPENDIX}
+%
+%   or
+%
+% \begin{APPENDICES}
+% \section{<Title of Section A>}
+% \section{<Title of Section B>}
+% etc
+% \end{APPENDICES}
```

Comment:
We could show an example of this in the skeleton Rmd using Custom blocks https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html

`````markdown
::: {.APPENDIX latex="{General appendix}"}

Some content for the appendix

:::
`````

or 

````markdown
::: {.APPENDICES latex=true}

# Section A

Content

# Section B

Content

:::
````

We could write in the skeleton`to show both example

`````markdown
::: {.APPENDIX latex="{General appendix}"}

Some content for the appendix

Note that if there are more than one appendix - several unrelated sections, one should use `APPENDICES` instead. 

````markdown

::: {.APPENDICES latex=true}

# Section A

Content

# Section B

Content

:::
````
::: 
````